eclipse-scout-chart/src/chart/ChartJsRenderer.ts

```diff
@@ -2794,7 +2794,7 @@ export class ChartJsRenderer extends AbstractChartRenderer {
     });
   }
 
-  protected _adjustAxisMaxMin(axis: LinearScaleOptions | RadialLinearScaleOptions, maxTicks: number, maxMinValue: Boundary) {
+  protected _adjustAxisMaxMin(axis: (CartesianChartScale | RadialChartScale) & { suggestedMax?: string | number; suggestedMin?: string | number; ticks?: (CartesianChartScale | RadialChartScale)['ticks'] & { stepSize?: number } }, maxTicks: number, maxMinValue: Boundary) {
```

Comment:
Maybe add a type for this as the line is very long and will be reformatted.

```suggestion
 protected _adjustAxisMaxMin(axis: AxisWithMaxMin, maxTicks: number, maxMinValue: Boundary) {
```

using
```
export type AxisWithMaxMin = (CartesianChartScale | RadialChartScale) & {
  suggestedMax?: string | number;
  suggestedMin?: string | number;
  ticks?: (CartesianChartScale | RadialChartScale)['ticks'] & {
    stepSize?: number;
  };
};
```